src/API/Management/GenericResource.php

```diff
@@ -106,6 +106,7 @@ protected function checkInvalidPermissions(array $permissions)
         }
 
         foreach ($permissions as $permission) {
+            
```

Comment:
typo?

Reply:
It's iterating through an array, the `$permission` is the current array element.